src/app/assumptions-and-limitation/assumptions-and-limitation.component.ts

```diff
@@ -82,16 +87,16 @@ export class AssumptionsAndLimitationComponent implements AfterContentInit {
   }
 
   public ngAfterContentInit(): void {
-    this.assumptionsLimitation.nativeElement.focus();
+    this.assumptionsLimitation.nativeElement.focus({ preventScroll: true });
   }
 
-  public onClose(hasFocus = true): void {
-    this.closeEvent.emit(hasFocus);
+  public onClose(focusOpenButton: boolean, scrollToOpenButton: boolean): void {
```

Comment:
Would it be possible to have this function use the `AssumptionsLimitationCloseEvent` as its parameter? Fine if it's awkward or doesn't work in the html, it's just a bit confusing seeing (true, false) or (true, true) with no context.

Reply:
I've reverted the function signature to `onClose(hasFocus: boolean = true)` now that the event has been reverted to just consist of the `hasFocus` boolean.